source/lib/predicates/array.ts

```diff
@@ -104,16 +106,6 @@ export class ArrayPredicate<T = any> extends Predicate<T[]> {
 		});
 	}
 
-	/**
-	 * Test an array to be not empty.
-	 */
-	get nonEmpty() {
```

Comment:
I don't think we want to remove these. They're still useful. `.not` applies to all the following modifiers, so it's still useful to be able to `ow.string.numeric.nonEmpty`, since `ow.string.not.numeric.empty` means something else.

Reply:
You're right, I missed the fact that `.not` applies to all the following modifiers, thanks!